Gemfile

```diff
@@ -28,61 +25,19 @@ if RUBY_VERSION >= '2.4.0'
   gem 'json', '>= 2.0.2'
 end
 
-if RUBY_VERSION < '1.9'
-  gem 'ffi', '< 1.9.19' # ffi dropped Ruby 1.8 support in 1.9.19
-else
-  gem 'ffi', '~> 1.9.25'
-end
-
-if RUBY_VERSION >= '2.0.0'
-  gem 'rake', '>= 10.0.0'
-elsif RUBY_VERSION >= '1.9.3'
-  gem 'rake', '< 12.0.0' # rake 12 requires Ruby 2.0.0 or later
-else
-  gem 'rake', '< 11.0.0' # rake 11 requires Ruby 1.9.3 or later
-end
+gem 'ffi', '~> 1.9.25'
 
-# Version 3 of mime-types 3 requires Ruby 2.0
-if RUBY_VERSION < '2.0.0'
-  gem 'mime-types', '< 3'
-end
+gem 'rake', '>= 10.0.0'
 
+gem 'mime-types', '< 3'
 
-# Capybara versions that support RSpec 3 only support RUBY_VERSION >= 1.9.3
-if RUBY_VERSION >= '1.9.3'
-  if /5(\.|-)[1-9]\d*/ === RAILS_VERSION || "master" == RAILS_VERSION
-    gem 'capybara', '~> 2.13', :require => false
-  else
-    gem 'capybara', '~> 2.2.0', :require => false
-  end
-end
+gem 'capybara', '~> 2.13', :require => false
 
-# Rack::Cache 1.3.0 requires Ruby >= 2.0.0
-gem 'rack-cache', '< 1.3.0' if RUBY_VERSION < '2.0.0'
-
-if RUBY_VERSION < '1.9.2'
-  gem 'nokogiri', '~> 1.5.0'
-elsif RUBY_VERSION < '1.9.3'
-  gem 'nokogiri', '1.5.2'
-elsif RUBY_VERSION < '2.1.0'
-  gem 'nokogiri', '1.6.8.1'
-else
-  gem 'nokogiri', '1.8.5'
-end
+gem 'nokogiri', '1.8.5'
 
-if RUBY_VERSION <= '1.8.7'
-  # cucumber and gherkin require rubyzip as a runtime dependency on 1.8.7
-  # Only < 1.0 supports 1.8.7
-  gem 'rubyzip', '< 1.0'
-else
-  gem "rubyzip", '>= 1.2.2'
-end
+gem "rubyzip", '>= 1.2.2'
```

Comment:
Most of these `if`s were to force older versions, with `else`s to suppress warnings about vulnerabilities its worth experimenting how many we can now just leave to bundler to figure out.

Reply:
I might change this to `~> 1.2` and let bundler figure it out.